src/main/java/io/zipcoder/Student.java

```diff
@@ -99,7 +136,15 @@ public boolean changeScoreForExam(int examNum, int newScore){
      * If they haven't taken any, be nice and give them 100.0.
      * @return The average for all the exams a student has taken.
      */
-    public double getAverage() {
+    public Double getAverage() {
+        double average = 0.0;
+        if (this.examsTaken > 0){
+            for (Double score: testScores) {
+                average += (score * 10);
+            }
+            DecimalFormat grading = new DecimalFormat("#.##");
+            return Double.parseDouble(grading.format((average/10)/examsTaken));
+        }
```

Comment:
still formatting to a string to parse right away, there are more optimal ways to do this